modules/admin_manual/pages/enterprise/metrics/metrics.adoc

```diff
@@ -0,0 +1,101 @@
+= Metrics
+:toc: right
+
+== Introduction
+
+The https://marketplace.owncloud.com/apps/metrics[Metrics App] provides a building block for monitoring and reporting of ownCloud Server. For simple use cases, it ships with an integrated dashboard that summarizes information about users, storage as well as shares and allows exporting it to a CSV file. Additionally, it adds a Metrics HTTP API endpoint to ownCloud Server, which can be used to obtain the Metrics data in regular intervals. Thus, more sophisticated analysis and visualizations can be conducted.
+
+The Metrics data are provided as snapshot values in the JSON format and are optimized to be consumed by professional data analyzers (like Splunk, ELK or Prometheus/Grafana) to collect statistics, derive visualizations and to set alerts for certain events of interest. They can be perfectly combined with the ownCloud Audit Logs (provided by the https://marketplace.owncloud.com/apps/admin_audit[Auditing App]) to gather time series data and to create a monitoring and reporting engine for ownCloud.
+
+Specifically, the extension adds:
+- an API endpoint which allows querying snapshot values of the system data as well as per-user data
+- an API endpoint for downloading the data in the CSV format
+- a dashboard that displays the snapshot data in the ownCloud Web UI and offers a CSV download (accessible by ownCloud administrators via the app launcher)
+
+TIP: Please note: If you're operating very large instances (users/files/shares,) it is recommended to use a special setup in order to not put the production database under huge load when gathering the values. For this, please replicate your installation (application server + read-only database) and install/use the Metrics App on the replica.
+
+The following data is available:
+
+**System data**
```

Comment:
```suggestion
.**System data**
```
You need to have a dot at the beginning to tell that the list below has a headline.
Else the whole thing will render as long line...